src/lucky/context_extensions.cr

```diff
@@ -1,26 +1,24 @@
 class HTTP::Server::Context
   DEBUG_COLOR = :green
-  setter cookies : Lucky::CookieJar?
-  setter session : Lucky::SessionCookie?
-  setter flash : Lucky::FlashStore?
-
   getter debug_messages : Array(String) = [] of String
   property? hide_from_logs : Bool = false
 
+  @_cookies : Lucky::CookieJar?
+
   def cookies
-    @cookies ||= Lucky::Cookies::Processors::Encryptor.read(
-      from: request
-    )
+    @_cookies ||= Lucky::CookieJar.from_request_cookies(request.cookies)
```

Comment:
Thanks! Originally we were thinking of having different processors but since we decided to always sign and encrypt, I think We likely won’t have other processors so I moved it into the CookieJar. What do you think?

Reply:
I do like the processors (or adapters or whatever they should be called) but I don't have an _actual_ use case for them. Easy enough to add them back. This is much simpler to work with for now.